lib/vagrant-openshift/action/install_origin_rhel7.rb

```diff
@@ -58,6 +58,7 @@ def call(env)
 ADD vars/* /etc/yum/vars/
 ADD repos/* /etc/yum.repos.d/
 ADD certs/* /var/lib/yum/
+ADD keys/* /etc/pki/rpm-gpg/
```

Comment:
This is actually exactly what I was talking about on the call - we're pulling later from repos that use these public keys for GPG package checks... but didn't have the keys